src/main/kotlin/uk/gov/gdx/datashare/models/TestEvent.kt

```diff
@@ -1,8 +1,10 @@
 package uk.gov.gdx.datashare.models
 
 import com.fasterxml.jackson.annotation.JsonInclude
+import io.swagger.v3.oas.annotations.media.Schema
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@Schema(description = "Test Event")
```

Comment:
If we do this, the test event will show in swagger right? Is there a way we can hide this one as its for testing only?

Reply:
Will check,